prow/deck/jobs/jobs.go

```diff
@@ -62,6 +62,7 @@ type Job struct {
 	State       string            `json:"state"`
 	Description string            `json:"description"`
 	URL         string            `json:"url"`
+	Path        string            `json:"path"`
```

Comment:
This is only useful for downstream consumers, right? Since we have the library in Prow to calculate this given a ProwJob. Why not expose an endpoint on Deck or something that gives you the GCS path for a job? Furthermore, _IF_ we can 100% know the path, we use it to create the URL, so this seems like it duplicates data?

Reply:
The duplication was intended to be temporary (going along with Spyglass and Gubernator co-existing for now), but it's probably best not to make that kind of thing an API change. So I'll undo most of the changes in that last commit and keep using the URL.